be/src/exec/workgroup/scan_task_queue.h

```diff
@@ -18,19 +19,26 @@ struct ScanTask {
     using WorkFunction = std::function<void(int)>;
 
     ScanTask() : ScanTask(nullptr, nullptr) {}
+    explicit ScanTask(WorkFunction work_function) : workgroup(nullptr), work_function(std::move(work_function)) {}
     ScanTask(WorkGroupPtr workgroup, WorkFunction work_function)
             : workgroup(std::move(workgroup)), work_function(std::move(work_function)) {}
     ~ScanTask() = default;
 
-    // Disable copy constructor and assignment.
-    ScanTask(const ScanTask&) = delete;
-    ScanTask& operator=(const ScanTask&) = delete;
+    DISALLOW_COPY(ScanTask);
     // Enable move constructor and assignment.
     ScanTask(ScanTask&&) = default;
     ScanTask& operator=(ScanTask&&) = default;
 
+    bool operator<(const ScanTask& rhs) const { return priority < rhs.priority; }
+    ScanTask& operator++() {
+        priority += 2;
```

Comment:
Why `++` means `priority += 2` ?

Reply:
The payload of tasks is `BlockingPriorityQueue`. After taking 512 tasks from the queue, the priority of remaining tasks in the queue will be upgrade 2.